azerothcore/azerothcore-wotlk
# fix(Docker): Persistent volumes for ac-database

(Docker) Persistent volume for ac-database

## CHANGES PROPOSED:
-  Added persistent volume for container `ac-database`


## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/2946


## TESTS PERFORMED:
- Ran `docker-compose config` to ensure syntax is right
- Started the containers
- Do some changes in the DB
- Stopped the containers with `docker-compose down`  which deletes the container
- Started the container and the data persisted



## HOW TO TEST THE CHANGES:

- Build AzerothCore in docker
- Start the containers
- Type `docker-compose down` which will remove the containers
- Start the containers again with `docker-compose up -d` and check database to see if data persisted.

## KNOWN ISSUES AND TODO LIST:

- None


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
